### ISSUE TITLE
Zigbee2mQTT wont't start after upgrade to 1.30.2

### ISSUE THREAD

**Issue by Flappiee** [Koenkk/zigbee2mqtt#16965]
### What happened?

Upgraded Zigbee2QTT to 1.30.2

Now Zigbee2MQTT isn't starting. Starting manual says:
administrator@HOMEASSISTANT1:/opt/zigbee2mqtt$ npm start

> zigbee2mqtt@1.30.2 start
> node index.js

/opt/zigbee2mqtt/node_modules/rimraf/dist/cjs/src/ignore-enoent.js:15
        if (er?.code !== 'ENOENT') {
               ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/opt/zigbee2mqtt/node_modules/rimraf/dist/cjs/src/rimraf-posix.js:14:28)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
npm ERR! code 1
npm ERR! path /opt/zigbee2mqtt
npm ERR! command failed
npm ERR! command sh -c node index.js

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/administrator/.npm/_logs/2023-03-08T19_11_50_003Z-debug.log
administrator@HOMEASSISTANT1:/opt/zigbee2mqtt$

### What did you expect to happen?

After upgrade complete, i expected a started Zigbee2MQTT instance

### How to reproduce it (minimal and precise)

Upgrade to latest version

### Zigbee2MQTT version

1.30.2

### Adapter firmware version

n/a

### Adapter

Zig-a-Zig-ah!

**Comment by Flappiee** [comment]
I'm running Debian and that is on node version v12.22.12.
Is that a problem?

**Comment by Flappiee** [comment]
Upgraded nodeje to version 18 LTS and problem is fixed.

**Comment by frfstu** [comment]
I'm update z2m 1.30.2 to!  Perhaps my torment is connected just not with the firmware of the stick, but with this one ?!

at com.zsmartsystems.zigbee.dongle.ember.EmberNcp.getConfiguration(EmberNcp.java:457) ~[?:?]

at com.zsmartsystems.zigbee.dongle.ember.internal.EmberStackConfiguration.getConfiguration(EmberStackConfiguration.java:72) ~[?:?]

at com.zsmartsystems.zigbee.dongle.ember.ZigBeeDongleEzsp.initialize(ZigBeeDongleEzsp.java:439) ~[?:?]

at com.zsmartsystems.zigbee.ZigBeeNetworkManager.initialize(ZigBeeNetworkManager.java:418) ~[?:?]

at org.openhab.binding.zigbee.handler.ZigBeeCoordinatorHandler.initialiseZigBee(ZigBeeCoordinatorHandler.java:431) ~[?:?]

at org.openhab.binding.zigbee.handler.ZigBeeCoordinatorHandler.lambda$2(ZigBeeCoordinatorHandler.java:557) ~[?:?]

at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]

at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]

at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[?:?]

at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]

at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]

at java.lang.Thread.run(Thread.java:829) [?:?]

I wrote more here. https://github.com/xsp1989/zigbeeFirmware/issues/37 

But this version works great with my previous coordinator zStack3x0 with firmware 20220219!

Help me to understand! I have 2 coordinators and they both do not work fully! Coordinator zStack3x0 with firmware 20220219 only works with Zigbee2MQTT (in this case, it is impossible to create automations, since this coordinator is not defined by the system as a whole), and ZB-GW04 v1.2 with firmware ncp-uart-sw_6.10.3_115200.gbl & later only works with zigbee bindings (constantly losing connection)!

**Comment by junosuarez** [comment]
The problem in the original issue is the old nodejs version. Upgrading to a newer nodejs is the solution.

**Comment by frfstu** [comment]
> The problem in the original issue is the old nodejs version. Upgrading to a newer nodejs is the solution.
On my system nodejs is already the newest version (16.19.1-deb-1nodesource1).

**Comment by junosuarez** [comment]
@frfstu your issue is different than the original one, since your stacktrace is showing Java code, not zigbee2mqtt code. If your coordinator's firmware won't run properly, or the coordinator is otherwise unavailable, then zigbee2mqtt won't start up.

**Comment by frfstu** [comment]
> @frfstu your issue is different than the original one, since your stacktrace is showing Java code, not zigbee2mqtt code. If your coordinator's firmware won't run properly, or the coordinator is otherwise unavailable, then zigbee2mqtt won't start up.

The fact is that this problem occurs only after installing and activating zigbee2mqtt! In my case, when installing the zigbee2mqtt service in openhab, the data of the latest published version is loaded, and as far as I understand (judging by the number of similar requests about the incorrect operation of zigbee2mqtt after the update), this problem is widespread among everyone who somehow updated zigbee2mqtt to the current version, and that's all these problems are somehow related to Java!

**Comment by frfstu** [comment]
> @frfstu your issue is different than the original one, since your stacktrace is showing Java code, not zigbee2mqtt code. If your coordinator's firmware won't run properly, or the coordinator is otherwise unavailable, then zigbee2mqtt won't start up.

You better tell me how to install an alternative (previous, stable) commit zigbee2mqtt in openhab via the console! That would make a lot more sense! Before I needed to install another coordinator (to test its operation), I installed version zigbee2mqtt 1.29.2 dated 01/14/2023 with which the smart home server worked stably without rebooting until the new coordinator was tested on 03/08/2023.
